mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
 function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
```

Comment:
Hi @nfarah22, 
The code in `1-syntax-errors.js` is still throwing a Syntax Error at the moment. Think you might need to double check this file.